manual/sphinx/user_docs/running_bout.rst

```diff
@@ -301,153 +245,149 @@ over this see if there are any important warnings or errors. Each
 
 This says that some run-time checking of values is enabled, that the
 code will try to catch segmentation faults to print a useful error, that
-NetCDF files are supported, but that the parallel flavour isn’t.
+NetCDF files are supported, but that the parallel flavour isn’t. The
+compilation flags are printed, which can be useful for checking if a
+run was built with optimisation or debugging enabled. These flags can
+be quite long, so we've truncated them in the snippet above.
 
-The processor number comes next::
+The complete command line is printed (excluding any MPI options)::
 
-    Processor number: 0 of 1
+        Command line options for this run : ./conduction nout=1
 
-This will always be processor number ’0’ on screen as only the output
-from processor ’0’ is sent to the terminal. After this the core BOUT++
-code reads some options::
-
-            Option /nout = 50 (data/BOUT.inp)
-            Option /timestep = 100 (data/BOUT.inp)
-            Option /grid = slab.6b5.r1.cdl (data/BOUT.inp)
-            Option /dump_float = true   (default)
-            Option /non_uniform = false (data/BOUT.inp)
-            Option /restart = false  (default)
-            Option /append = false  (default)
-            Option /dump_format = nc (data/BOUT.inp)
-            Option /StaggerGrids = false  (default)
+After this the core BOUT++ code reads some options::
+
+    Reading options file data/BOUT.inp
+        Option nout = 100 (data/BOUT.inp) overwritten with:
+            nout = 1 (Command line)
+    Writing options to file data/BOUT.settings
+
+    Getting grid data from options
+        Option mesh:type = bout (default)
+        Option mesh:StaggerGrids = 0 (default)
+        Option mesh:maxregionblocksize = 64 (default)
+        Option mesh:calcParallelSlices_on_communicate = 1 (default)
+        Option mesh:ddz:fft_filter = 0 (default)
+        Option mesh:symmetricGlobalX = 1 (default)
+        Option mesh:symmetricglobaly = true (data/BOUT.inp)
 
 This lists each option and the value it has been assigned. For every
 option the source of the value being used is also given. If a value had
 been given on the command line then ``(command line)`` would appear
 after the option.::
 
-    Setting X differencing methods
-            First       :  Second order central (C2)
-            Second      :  Second order central (C2)
-            Upwind      :  Third order WENO (W3)
-            Flux        :  Split into upwind and central (SPLIT)
-    Setting Y differencing methods
-            First       :  Fourth order central (C4)
-            Second      :  Fourth order central (C4)
-            Upwind      :  Third order WENO (W3)
-            Flux        :  Split into upwind and central (SPLIT)
-    Setting Z differencing methods
-            First       :  FFT (FFT)
-            Second      :  FFT (FFT)
-            Upwind      :  Third order WENO (W3)
-            Flux        :  Split into upwind and central (SPLIT)
-
```

Comment:
It is a bit of a shame to have lost this output. Should we reinstate it?

Reply:
:+1: I agree would be nice to reinstate it.